### PR TITLE
Typehud Keymap

### PR DESCRIPTION
**Pull request by christanaka**
Hey Jay!

This keymap features a wpm graph, input bar, and a keyboard matrix. You can configure the graph and matrix via `config.h` as documented in the README. I'll also open a PR for the snap if I find someone to test it!

### CODE REVIEW

File: keymaps/typehud/keymap.c [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  [_VIA3] = LAYOUT(
+                  ___, ___, ___,
+  ___, ___, ___, ___, ___, ___,
+  ___, ___, ___, ___, ___, ___,
+  ___, ___, ___, ___, ___, ___,
+  ___, ___, ___, ___, ___, ___
+  ),
+};
+
+#ifndef OLED_ROTATE
```
Comment (jaygreco) [inline keymaps/typehud/keymap.c:62]: I understand the intent here (add a default) but I think this is redundant and might confuse people who are editing. I'd recommend removing the define here and leaving the one in `config.h`.

**Comment by jaygreco** [comment]
As always, thanks so much for this awesome keymap and the time you spent putting together the PR! Only one piece of feedback from me and then I'm happy to merge :D

**Comment by christanaka** [comment]
Will make that change! I also am in the process of adding additional logic for split keyboards which I will also add just so that the source for `typehud.h` is consistent across all repos!

**Comment by jaygreco** [comment]
Awesome, no rush :D

**Comment by christanaka** [comment]
Keymap updated! It also includes the changes for split keyboards!

**Pull request merged**